.coderabbit.yaml

```
@@ -1,212 +1,114 @@
+# ============================================================================
+# 기본 리뷰 설정
+# ============================================================================
 reviews:
-  # 자동 리뷰 활성화
-  auto_review: true
-  
-  # 리뷰 대상 파일 패턴
+  # 자동 리뷰 활성화 여부
+  auto_review:
+    enabled: true    # PR 생성/업데이트 시 자동으로 리뷰 실행
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)